core/utils/metadata-utils/src/main/java/datawave/query/util/NumShardMetadataHelper.java

```diff
@@ -0,0 +1,57 @@
+package datawave.query.util;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Map;
+
+import org.apache.accumulo.core.client.AccumuloClient;
+import org.apache.accumulo.core.client.AccumuloException;
+import org.apache.accumulo.core.client.AccumuloSecurityException;
+import org.apache.accumulo.core.client.Scanner;
+import org.apache.accumulo.core.client.TableExistsException;
+import org.apache.accumulo.core.client.TableNotFoundException;
+import org.apache.accumulo.core.client.admin.TableOperations;
+import org.apache.accumulo.core.data.Key;
+import org.apache.accumulo.core.data.Range;
+import org.apache.accumulo.core.data.Value;
+import org.apache.accumulo.core.security.Authorizations;
+import org.apache.hadoop.io.Text;
+import org.apache.log4j.Logger;
+
+public class NumShardMetadataHelper {
+
+    private static final Logger log = Logger.getLogger(NumShardMetadataHelper.class);
+
+    public static ArrayList<String> getNumShardEntries(AccumuloClient accumuloClient, String metadataTableName, Text numShards, Text numShardsCF)
```

Comment:
Please look at the metadata helper for examples:
1) should not be static
2) add caching like we do for a mess of the other calls
3) The auths should use the "root auths" like we do on other scanners against the metadata table.